packages/block-library/src/image/edit.native.js

```diff
@@ -813,6 +832,13 @@ export class ImageEdit extends Component {
 							} ) => {
 								return (
 									<View style={ imageContainerStyles }>
+										{ isFetchingImage && (
+											<View
+												style={ styles.image__loading }
+											>
+												<ActivityIndicator animating />
+											</View>
+										) }
```

Comment:
I noticed that if the Image block is empty (i.e. it's showing the placeholder), the loading indicator is not being rendered when waiting for the Image's size request. We might consider rendering this outside the `renderContent` render function to display it in all cases.

Reply:
@fluiddot Good catch! I've added the loading indicator to the [MediaPlaceholder container](https://github.com/WordPress/gutenberg/pull/40334/commits/ee61ccb3b1eabce9dbc32fbd6c47cc64277002e7#diff-7126ae03000a022bcefa0d2a5fb03b6aca56ed4047e3a49d2131babf229cf6c1R762-R768), which should only display when no `url` attribute is present. When a `url` attribute is already present, the loading indicator from `getImageComponent` will be used.

https://user-images.githubusercontent.com/643285/168992453-feb099cd-a660-4bc3-bb52-ef2b06abc43f.mov

Reply:
Also, the [Static Analysis check](https://github.com/WordPress/gutenberg/runs/6485425966?check_suite_focus=true)s (Linting, License, Type checks...) are not failing locally for me. On some CI builds, the pass, and others, they are failing. I am looking for anything that may be causing them to fail on CI. 🤔

Reply:
:wave: @derekblank ! That's really weird that the checks are sometimes passing and sometimes failing. In case you can't see the CI logs, these are the errors it's reporting on the latest failed run:

```
/home/runner/work/gutenberg/gutenberg/packages/block-library/src/image/edit.native.js
[1]   763:12  error  Replace `⏎↹↹↹↹↹↹↹style={·styles.image__loading·}⏎↹↹↹↹↹↹` with `·style={·styles.image__loading·}`  prettier/prettier
[1]   768:9   error  Delete `↹`      
```

Reply:
Thanks for the pointer, @mchowning ! That linter command wasn't failing (or wasn't running) with my pre-commit or pre-push actions locally -- I had to run `npm run lint` specifically to see the failure. I think that was my issue with some of the previous Static Analysis failures as well. It's all fixed up now. 👍